BlackJack.py

```diff
@@ -0,0 +1,77 @@
+jogadores = {}
+jogadores_inativos = []
+
+def cria_baralho():
+    baralho = (['A'] + list(range(2,11)) + ['J', 'Q', 'K'])*4
+    return baralho
+
+def sorteio(baralho):
+    import random
```

Comment:
O ideal é importar todos os módulos no início do código. Em alguns casos, principalmente em arquivos do Jupyter, é até comum o pessoal importar no meio do notebook, no momento em que o módulo será utilizado. Porém, o problema, nesse caso, é que o módulo está sendo importado várias vezes, sempre que a função `sorteio` é chamada.